components/storage/blockdevice/COMPONENT_SD/config/mbed_lib.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "sd",
     "config": {
-        "SPI_CS": "NC",
-        "SPI_MOSI": "NC",
-        "SPI_MISO": "NC",
-        "SPI_CLK": "NC",
+        "SPI_CS": "SPI_PERSISTENT_MEM_CS",
+        "SPI_MOSI": "SPI_MOSI",
+        "SPI_MISO": "SPI_MISO",
+        "SPI_CLK": "SPI_SCK",
         "DEVICE_SPI": 1,
```

Comment:
It seems that MBED_CONF_SD_DEVICE_SPI is not used

Reply:
do you mean MBED_CONF_SD_SPI